shell/platform/windows/window_win32.cc

```diff
@@ -556,6 +562,24 @@ HWND WindowWin32::GetWindowHandle() {
   return window_handle_;
 }
 
+float WindowWin32::GetScrollOffsetMultiplier() {
+  return scroll_offset_multiplier_;
+}
+
+void WindowWin32::UpdateScrollOffsetMultiplier() {
+  UINT lines_per_scroll;
+  // Get lines per scroll wheel value from Windows
+  if (SystemParametersInfo(SPI_GETWHEELSCROLLLINES, 0, &lines_per_scroll, 0) ==
+      0) {
+    lines_per_scroll = kLinesPerScrollWindowsDefault;
+  }
```

Comment:
What do you think of simplifying to this?

```suggestion
    lines_per_scroll = kLinesPerScrollWindowsDefault;
    SystemParametersInfo(SPI_GETWHEELSCROLLLINES, 0, &lines_per_scroll, 0);
```

Reply:
Makes sense, added.